pkg/globalmanager/controllers/incrementallearning/incrementallearningjob.go

```diff
@@ -806,11 +838,35 @@ func (c *Controller) createInferPod(job *sednav1.IncrementalLearningJob) error {
 		"LC_SERVER": c.cfg.LC.Server,
 	}
 
+	modelHotUpdate := job.Spec.DeploySpec.Model.HotUpdateEnabled
+	getModelHotUpdateConfigFile := func(prefix string) string {
+		return strings.ToLower(filepath.Join(prefix, job.Namespace, job.Kind, job.Name, runtime.ModelHotUpdateConfigFile))
+	}
+
+	if modelHotUpdate {
+		workerParam.Env["MODEL_HOT_UPDATE"] = "true"
```

Comment:
Add the modelHotUpdate-related environment variables logic into `worker.go` or new file of directory of `worker.go`, since they can be reused by other features if needed.

Reply:
fix it